tests/functional/aws-node-sdk/test/object/deleteMpu.js

```diff
@@ -46,7 +23,75 @@ describe('DELETE multipart', () => {
                     `Found unexpected statusCode ${statusCode}`);
                 assert.strictEqual(err, null,
                     `Expected no err but found ${err}`);
-                done(err);
+                callback(err);
+            });
+        }
+
+        it('on bucket that does not exist: should return NoSuchBucket',
```

Comment:
@rahulreddy : confirmed aws returns NoSuchBucket for region other than us-east-1:
```
$ aws s3api  abort-multipart-upload --bucket nonexistingbucket --key objectKey --upload-id nonexistingid --region us-west-1

An error occurred (NoSuchBucket) when calling the AbortMultipartUpload operation: The specified bucket does not exist
```